src/AWS.Deploy.Common/Recipes/MSPropertyRule.cs

```diff
@@ -1,23 +0,0 @@
-﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-// SPDX-License-Identifier: Apache-2.0
-
-using System.Collections.Generic;
-
-namespace AWS.Deploy.Common.Recipes
```

Comment:
and this also, aren't we using them?

Reply:
This class was not used at all. There were no references for it. I also ran the integ tests and they passed.